emulated_hue/api.py

```diff
@@ -663,6 +669,12 @@ async def __async_entity_to_hue(
                     const.HUE_ATTR_XY: entity_attr.get(
                         const.HASS_ATTR_XY_COLOR, [0, 0]
                     ),
+                    const.HUE_ATTR_HUE: entity_attr.get(
```

Comment:
I'm not so sure if setting all the values at once is a good idea. It's either xy, hs or ct not all at the same time.
We'll have to see if this doesn't bring us in trouble with some testing.

Reply:
Its actually required by the hue apps and is how the official bridge responds. Fixes the hue app not showing both color modes.